Solutions/Windows Server DNS/Data Connectors/template_ASimDnsActivityLogs.JSON

```diff
@@ -0,0 +1,80 @@
+{
+    "id": "ASimDnsActivityLogs",
```

Comment:
The id is misleading as the table is not intended to be used just by this connector. He can create problems in the future.